R/bootstrap-layout.R

```diff
@@ -516,7 +516,7 @@ splitLayout <- function(..., cellWidths = NULL, cellArgs = list()) {
   children <- children[childIdx]
   count <- length(children)
 
-  if (length(cellWidths) == 0 || is.na(cellWidths)) {
+  if (length(cellWidths) == 0 || any(is.na(cellWidths))) {
```

Comment:
This way there won't be a change in behavior for a case like `cellWidths=c("50%", NA, "50%")`

```suggestion
  if (length(cellWidths) == 0 || isTRUE(is.na(cellWidths))) {
```